otx/core/data/adapter/anomaly_dataset_adapter.py

```diff
@@ -77,150 +59,54 @@ def _prepare_anomaly_label_information(self) -> List[LabelEntity]:
             domain=self.domain,
             is_anomalous=True,
         )
-        return [normal_label, abnormal_label]
-
-    def get_otx_dataset(self) -> DatasetEntity:
-        """Get DatasetEntity."""
-        raise NotImplementedError()
 
-
-class AnomalyClassificationDatasetAdapter(AnomalyBaseDatasetAdapter):
-    """Anomaly classification adapter inherited from AnomalyBaseDatasetAdapter."""
+        label_information = super()._prepare_label_information(datumaro_dataset)
+        label_information["label_entities"] = [
+            normal_label if class_name.name == "good" else abnormal_label
+            for class_name in label_information["category_items"]
+        ]
+
+        return label_information
+
+    def _get_shapes_from_datumaro_item(self, datumaro_item):
+        height, width = datumaro_item.media.size
+        shapes: List[Annotation] = []
+        for ann in datumaro_item.annotations:
+            if ann.type == AnnotationType.label:
+                shapes.append(self._get_label_entity(ann))
+            elif ann.type == AnnotationType.mask:
+                datumaro_polygons = MasksToPolygons.convert_mask(ann)
+                for d_polygon in datumaro_polygons:
+                    shapes.append(self._get_polygon_entity(d_polygon, width, height))
+            elif ann.type == AnnotationType.bbox:
+                if self._is_normal_bbox(ann.points[0], ann.points[1], ann.points[2], ann.points[3]):
+                    shapes.append(self._get_normalized_bbox_entity(ann, width, height))
+        return shapes
 
     def get_otx_dataset(self) -> DatasetEntity:
-        """Convert DatumaroDataset to DatasetEntity for Anomaly classification."""
-        normal_label, abnormal_label = self._prepare_anomaly_label_information()
-        self.label_entities = [normal_label, abnormal_label]
+        """Convert DatumaroDataset to DatasetEntity for Anomaly segmentation."""
```

Comment:
Anomaly tasks maybe?

Reply:
yep, this is a leftover from development